Packs/SplunkPy/ReleaseNotes/2_5_0.md

```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+##### SplunkPy
+- Migrated from python 2.7 to python 3.10.
+- Upgraded the splunk-sdk to 1.7.2.
+- Removed the `use_requests_handler` parameter. If selected to use `proxy` or `insecure` parameters, the integration will use the requests handler.
+- Updated the Docker image to: *demisto/splunksdk-py3:1.0.0.37176*.
```

Comment:
please update according the decision to use the handler always 

also the name of this file and the json file should be 3_0_0

Reply:
changed